lib/griddler/amazon_s3_ses/adapter.rb

```diff
@@ -90,7 +90,15 @@ def multipart?
       end
 
       def text_part
-        force_body_to_utf_8_string(multipart? ? message.text_part.try(:body) : Sanitize.clean(message.try(:body)))
+        text = if multipart?
+          message.text_part.try(:body)
+        else
+          Sanitize.clean(message.try(:body))
+                  .gsub(/\r/, '')
+                  .gsub(/\n/, '')
+                  .gsub(/[ ]{3,}/, '')
+        end
+        force_body_to_utf_8_string(text)
```

Comment:
Super nit... Might be a little nicer to read if it was broken into two methods:

```
def text_part
  text = if multipart? ? message.text_part.try(:body) : sanitized_message_body
  force_body_to_utf_8_string(text)
end

def sanitized_message_body
  Sanitize.clean(message.try(:body))
          .gsub(/\r/, '')
          .gsub(/\n/, '')
          .gsub(/[ ]{3,}/, '')
end
```

Additionally, could also squash those gsubs into one since they're all being mapped to the same thing

```
string = "Some text \n \n \r lalala"
string.gsub(/\n|\r/, '-')
=> "asdsad - - - lalala"
```

Reply:
👍 Updated in https://github.com/retailzipline/griddler-amazon_s3_ses/pull/5/commits/b8e2857009b7ef6ab17cde4774c712f6ee73ed53